src/main/java/eu/accesa/teamview/security/JwtController.java

```diff
@@ -0,0 +1,49 @@
+package eu.accesa.teamview.security;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.security.authentication.AuthenticationManager;
+import org.springframework.security.authentication.BadCredentialsException;
+import org.springframework.security.authentication.DisabledException;
+import org.springframework.security.authentication.UsernamePasswordAuthenticationToken;
+import org.springframework.security.core.userdetails.UserDetails;
+import org.springframework.web.bind.annotation.CrossOrigin;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@CrossOrigin
+public class JwtController {
+
+    private static final Logger logger = LoggerFactory.getLogger(JwtController.class);
+    @Autowired
+    private JwtUserDetailsService jwtUserDetailsService;
+
+    @Autowired
+    private AuthenticationManager manager;
+
+    @Autowired
+    private TokenManager tokenManager;
+
+    @PostMapping("/login")
+    public ResponseEntity<JwtResponseModel> createToken(@RequestBody JwtRequestModel jwtRequestModel) throws Exception {
```

Comment:
This controller method is doing a bit too much. I would suggest to move the logic into a JwtService.